manifests/07-downloads-deployment.yaml

```diff
@@ -89,10 +89,14 @@ spec:
               os.mkdir(arch)
               for operating_system in ['linux', 'mac', 'windows']:
                   os.mkdir(os.path.join(arch, operating_system))
+                  target_path = os.path.join(arch, operating_system, 'LICENSE')
+                  os.symlink('/usr/share/openshift/LICENSE', target_path)
```

Comment:
Is this path also valid for windows ? not really sure about that...

Reply:
`target_path` is path after FQDN, so URL is `<scheme>://<FQDN>/<taret_path>`.
For windows, it is `https://<FQDN>/adm64/windows/LICENSE`.